src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import * as api from "../api";
+
+export default function ArticleComments({ comments, setComments }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    api
+      .fetchComments(article_id)
+      .then((comments) => {
+        setComments(comments);
+      })
+      .then(() => {
+        setIsLoading(false);
+      });
+  }, [article_id, setComments]);
+
+  return isLoading ? (
+    <p>Loading comments...</p>
+  ) : (
+    <div className="comments-card">
+      <ul className="comments-list">
+        {comments.map((comment) => {
+          return (
+            <div key={comment.comment_id} className="comment-list-card">
+              <li>
```

Comment:
The div wrapping the li here doesn't do anything apart from breaking the semantic flow of your html. Take it out and apply the key and className to the li directly.